datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/config.py

```diff
@@ -141,6 +150,24 @@ def config(ctx, check, sync, verbose):
         abort()
 
 
+def validate_default_template(spec_file):
+    init_config_default = False
+    instances_default = False
+    if 'template: init_config' not in spec_file and 'template: instances' not in spec_file:
```

Comment:
```suggestion
    if 'template: init_config' not in spec_file or 'template: instances' not in spec_file:
```
Don't you need both `template: init_config` and `template: instances` in `spec.yaml`?

Reply:
hmm actually, I'm trying to be strict about which integrations we're allowing to skip default template validation. It could be possible that the `template: instances` and `template: init_config` was left out.